Task_TheFlexGame/index.css

```diff
@@ -1 +1,61 @@
 /* Start coding below for FLEX GAME TASK*/
+
+.main {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.header {
+  background-color: red;
+  padding-bottom: 20px;
```

Comment:
Consider using the shorthand padding property to combine the padding-left and padding-right styles.